qdrant-landing/themes/qdrant/static/css/blocks/_documentation.scss

```
@@ -39,6 +39,11 @@
     h6 {
       font-size: 1rem;
     }
+
+    // Correct spacing of unordered list in table cells
+    & table tr td ul {
+        margin: 0 !important;
```

Seems to work without `!important`

```suggestion
        margin: 0;
```

---

You're right. Left this behind accidentally. Thanks!

---

Removed in 97d593226272ef9a4835d02c76b4035fbb7d02a2. :+1: